roles/openshift_facts/library/openshift_facts.py

```diff
@@ -663,6 +663,17 @@ def set_nodename(facts):
     return facts
 
 
+def set_schedulable(facts):
```

Comment:
I was confused earlier.  We don't want to add this back to openshift_facts.

If masters not default to schedulable, we don't need this and we can use your previous iteration.

Reply:
Okay, will revert that with a comment there